internal/apiutil/transport.go

```diff
@@ -47,7 +46,9 @@ func LoggingErrorEncoder(logger logger.Logger, enc kithttp.ErrorEncoder) kithttp
 			errors.Contains(err, ErrInvalidAPIKey),
 			errors.Contains(err, ErrMaxLevelExceeded),
 			errors.Contains(err, ErrBootstrapState),
-			errors.Contains(err, errors.ErrInvalidQueryParams):
+			errors.Contains(err, ErrInvalidQueryParams),
+			errors.Contains(err, ErrUnsupportedContentType),
+			errors.Contains(err, ErrNotFoundParam):
```

Comment:
I'd keep this error in apituil but I'd not log it since it's only used to know if the boolean parameter is present in the query.